changes/ocd-4513.sql

```diff
@@ -0,0 +1,434 @@
+	'https://app.powerbi.com/view?r=eyJrIjoiZjZhYzg5OTgtMzkxYy00YzIzLWE2MzItMDI0YzcwYTRiNGNkIiwidCI6IjMwN2QyMTJhLWZiODYtNDgwNy04NGRkLTg2Nzc2OWI4MDQyYSIsImMiOjF9',
+	'2500px',
+	2,
+	-1
+where not exists (
+	select * from openchpl.report_metadata where environment = 'DEV' and report_key = 'SurveillanceStatistics'
+);
+
+insert into openchpl.report_metadata (environment, title, report_key, report_group, url, height, display_order, last_modified_user)
+select 'QA', 
+	'Surveillance Statistics',
+	'SurveillanceStatistics', 
+	'dashboard', 
+	'https://app.powerbi.com/view?r=eyJrIjoiZDk2MDM0MWUtOGFjOC00MzY5LTkzNzUtNWM3ZDljMzE4M2E1IiwidCI6IjMwN2QyMTJhLWZiODYtNDgwNy04NGRkLTg2Nzc2OWI4MDQyYSIsImMiOjF9',
+	'2500px',
+	2,
+	-1
+where not exists (
+	select * from openchpl.report_metadata where environment = 'QA' and report_key = 'SurveillanceStatistics'
+);
+
+insert into openchpl.report_metadata (environment, title, report_key, report_group, url, height, display_order, last_modified_user)
+select 'STG',
+	'Surveillance Statistics',
+	'SurveillanceStatistics', 
+	'dashboard', 
+	'https://app.powerbi.com/view?r=eyJrIjoiMGU4OGU2ZjgtOTMyNC00YzcxLWI2YTUtYTk2ZjU2ZTg0OWVjIiwidCI6IjMwN2QyMTJhLWZiODYtNDgwNy04NGRkLTg2Nzc2OWI4MDQyYSIsImMiOjF9',
+	'2500px',
+	2,
+	-1
+where not exists (
+	select * from openchpl.report_metadata where environment = 'STG' and report_key = 'SurveillanceStatistics'
+);
+
+
+insert into openchpl.report_metadata (environment, title, report_key, report_group, url, height, display_order, last_modified_user)
+select 'PROD', 
+	'Surveillance Statistics',
+	'SurveillanceStatistics', 
+	'dashboard', 
+	'https://app.powerbi.com/view?r=eyJrIjoiMThmM2FmYWMtNTgyMS00MDQ0LWJmNTYtYjJiNzdhODkxZGM1IiwidCI6IjMwN2QyMTJhLWZiODYtNDgwNy04NGRkLTg2Nzc2OWI4MDQyYSIsImMiOjF9',
+	'2500px',
+	2,
+	-1
+where not exists (
+	select * from openchpl.report_metadata where environment = 'PROD' and report_key = 'SurveillanceStatistics'
+);
+
+insert into openchpl.report_metadata (environment, title, report_key, report_group, url, height, display_order, last_modified_user)
+select 'DEV', 
+	'Criteria Attributes',
+	'CriteriaAttributes', 
+	'dashboard', 
+	'https://app.powerbi.com/view?r=eyJrIjoiMGI2MGMzMzUtYjk5My00ZDRhLTllOWMtMzRhMTFlZDE3ZjIxIiwidCI6IjMwN2QyMTJhLWZiODYtNDgwNy04NGRkLTg2Nzc2OWI4MDQyYSIsImMiOjF9',
+	'500px',
+	6,
+	-1
+where not exists (
+	select * from openchpl.report_metadata where environment = 'DEV' and report_key = 'CriteriaAttributes'
+);
+
+insert into openchpl.report_metadata (environment, title, report_key, report_group, url, height, display_order, last_modified_user)
+select 'QA', 
+	'Criteria Attributes',
+	'CriteriaAttributes', 
+	'dashboard', 
+	'https://app.powerbi.com/view?r=eyJrIjoiMGI2MGMzMzUtYjk5My00ZDRhLTllOWMtMzRhMTFlZDE3ZjIxIiwidCI6IjMwN2QyMTJhLWZiODYtNDgwNy04NGRkLTg2Nzc2OWI4MDQyYSIsImMiOjF9',
+	'500px',
+	6,
+	-1
+where not exists (
+	select * from openchpl.report_metadata where environment = 'QA' and report_key = 'CriteriaAttributes'
+);
+
```

Comment:
These have the same URLs, but none of the other reports have matching URLs. Should these match? Or should they be different? Or should some of the other ones match in some other way?